features/distributed/kv-store/blob/blob-shell/src/main/java/org/opennms/features/distributed/kvstore/blob/shell/BlobStoreGet.java

```diff
@@ -36,13 +36,13 @@
 import org.apache.karaf.shell.api.action.Command;
 import org.apache.karaf.shell.api.action.lifecycle.Reference;
 import org.apache.karaf.shell.api.action.lifecycle.Service;
-import org.opennms.features.distributed.kvstore.api.KeyValueStore;
+import org.opennms.features.distributed.kvstore.api.BlobStore;
 
-@Command(scope = "kvstore", name = "get", description = "Get a record from the key value store")
+@Command(scope = "blobstore", name = "get", description = "Get a record from the blob store decoded as a string")
```

Comment:
Can we prefix the shell commands with `opennms-`? (See [NMS-10413](https://issues.opennms.org/browse/NMS-10413))